docs/content/docs/libs/cep.md

```diff
@@ -1528,9 +1528,9 @@ The whole processing is done with event time.
 {{< tabs "573ac3c5-e8b9-4ffa-b7b6-e2db19611ff5" >}}
 {{< tab "Java" >}}
 ```java
-StreamExecutionEnvironment env = ...
+StreamExecutionEnvironment env = ...；
 
-DataStream<Event> input = ...
+DataStream<Event> input = ...；
```

Comment:
minor： Use `;` instead of `；`

Reply:
Sorry for the stupid problem, and it have been fixed.

Reply:
Hi @leonardBang, could you help me to comfirm